src/test/java/core/basesyntax/service/impl/FileWriterImplTest.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.FileWriter;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FileWriterImplTest {
+    public static final String WRITE_FILEPATH = "src/test/resources/write_test.csv";
+    public static final String EMPTY_FILEPATH = "";
+    public static final String TEST_STRING = "type,fruit,quantity" + System.lineSeparator()
+            + "b,banana,20";
+    private FileWriter writer;
+
+    @Before
+    public void setUp() {
+        writer = new FileWriterImpl();
+    }
+
+    @Test
+    public void writeData_ok() {
+        writer.writeData(TEST_STRING, WRITE_FILEPATH);
+        List<String> actual;
+        List<String> expected = List.of("type,fruit,quantity", "b,banana,20");
+        try {
+            actual = Files.readAllLines(Path.of(WRITE_FILEPATH));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read data from file " + WRITE_FILEPATH);
+        }
+        assertEquals(expected, actual);
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void writeToEmptyFilepath_notOk() {
+        writer.writeData(TEST_STRING, EMPTY_FILEPATH);
+    }
+
+    @After
+    public void tearDown() {
+        File file = new File(WRITE_FILEPATH);
+        file.delete();
```

Comment:
`Files.deleteIfExists(path)`
better use this method instead

Reply:
@aaananas, I fixed this. But what is unusual for me is that when I throw IOExeption in try-catch, IDE requires to 'throws IOException' outside also. Is it ok?